subprojects/testfx-core/src/main/java/org/testfx/robot/Motion.java

```diff
@@ -38,7 +38,7 @@
  * <p>
  * Traveling in a straight-line between a and b (that is, tracing the hypotenuse) is
  * {@code DIRECT}. Traveling first from a to c and then from c to b is {@code HORIZONTAL_FIRST}.
- * Traveling first from a to d and then from d to b is {@code VERTICAL_FIRST}. {@code DIRECT}
+ * Traveling first from a to d and then from d to b is {@code VERTICAL_FIRST}. {@code DEFAULT}

```

Comment:
I am not sure about this change. In terms of the enum itself there is a `DEFAULT` which has the semantic meaning "Whatever TestFX defines as the default way to move between two point.". `DIRECT` is currently the `DEFAULT`, but that could change in the future. Having the value of `DIRECT` means specifically to use direct (naively diagonal, although of course it could be only horizontal or only vertical depending on the start and end points) motion whereas `DEFAULT` means use the TestFX default which is *currently* `DIRECT` but could change!

Reply:
I'm confused. Are we on the same page that the docs currently say `DIRECT`, and this PR changes it to say `DEFAULT`?

Reply:
I was the one confused. Sorry about that, lol!